nixos/modules/services/networking/headscale.nix

```diff
@@ -313,21 +361,21 @@ in {
 
               allowed_domains = mkOption {
                 type = types.listOf types.str;
-                default = [ ];
+                default = [];
                 description = ''
                   Allowed principal domains. if an authenticated user's domain
                   is not in this list authentication request will be rejected.
                 '';
-                example = [ "example.com" ];
+                example = ["example.com"];
               };
 
               allowed_users = mkOption {
                 type = types.listOf types.str;
-                default = [ ];
+                default = [];
                 description = ''
                   Users allowed to authenticate even if not in allowedDomains.
                 '';
-                example = [ "alice@example.com" ];
+                example = ["alice@example.com"];
```

Comment:
I am not sure if nixfmt-rfc-style would add those back. Please use that package instead when changing formatting.